### PR TITLE
58604 update category

### PR DESCRIPTION
**Pull request by tanphan313**

### CODE REVIEW

File: app/views/admin/categories/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <%= f.label :title, t("category_title") %>
+  </div>
+
+  <div class="input-field col s12">
+    <%= f.text_area :description, class: "materialize-textarea", required: true %>
+    <%= f.label :title, t("category_description") %>
+  </div>
+
+  <%= f.submit t("submit"), class: "btn waves-effect waves-light" %>
+<% end %>
```
Comment (hellovietnam93) [inline app/views/admin/categories/_form.html.erb]: lack of end file

File: app/views/admin/categories/index.html.erb [hunk 2]
```diff
@@ -16,16 +16,20 @@
           </tr>
         </thead>
         <tbody>
-          <% @categories.each do |category| %>
+          <% @categories.each do |item| %>
```
Comment (hellovietnam93) [inline app/views/admin/categories/index.html.erb]: <% @categories.each do |category| %>

File: app/views/admin/categories/index.html.erb [hunk 3]
```diff
@@ -23,9 +23,13 @@
               </td>
               <td>
                 <%= link_to t("delete"), [:admin, category], method: :delete,
-                  data: {confirm: t("confirm")},
+                  data: { confirm: t("confirm") },
```
Comment (hellovietnam93) [inline app/views/admin/categories/index.html.erb]: remove space inside {}

File: app/controllers/admin/categories_controller.rb [hunk 4]
```diff
@@ -23,8 +25,29 @@ def destroy
...
+  def edit
+
+  end
+
+  def update
+    if @category.update_attributes category_params
+      flash[:success] = t "update_success"
+      redirect_to edit_admin_category_path @category
+    else
+      render :edit
```
Comment (vietmt) [inline app/controllers/admin/categories_controller.rb]: Flash error mess

**Pull request merged**